programs/openbook_v2/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
```

Comment:
in reality - if drift will upgrade solana-security-txt = "1.1.0" to 1.1.1 - most of the code in the crate will be irelevant, because openbook v2 could be imported directly

Reply:
but in the end is there any penalty - performance etc. for doing it this way?